conda_smithy/configure_feedstock.py

```diff
@@ -10,7 +10,11 @@
 import copy
 import hashlib
 import requests
-import json
+
+try:
```

Comment:
Can you please add a comment as to why this is needed, because it it pretty weird that this is triggered by requests

Reply:
I agree this error is weird and boring. I wasn't aware of it but after some googling, it seems to be annoying a lot of people (https://github.com/psf/requests/issues/4842#issuecomment-665502871 among others).

Reply:
Done.